src/styles/blocks/product-card.scss

```
@@ -0,0 +1,131 @@
+$productMarginY: 23px;
+$productMarginX: 24px;
+$productWidth: 200px;
+
+@function calcWidth ($cardWidth, $cardMarginX) {
+  @return ($cardMarginX * 8 + $cardWidth * 4);
+}
+
+@mixin fontSizeWeight($size, $weight: 400) {
+  font-size: $size;
+  font-weight: $weight;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.products {
+  display: flex;
+  width: calcWidth($productWidth, $productMarginX);
+  flex-wrap: wrap;
+
+}
+
+.page__products {
+  margin: (50px - $productMarginY) auto 72px;
+}
+
+.products__card {
+  margin: $productMarginY $productMarginX;
+}
+
+.product-card {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: $productWidth;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background-color: white;
```

don't forget to create variables for colors
create in all cases
```suggestion
  border: 1px solid $c-color;
  background-color: $c-white;
```